scripts/e2e-deploy.sh

```diff
@@ -168,7 +173,7 @@ done
 
 ### deploy application
 echo "Creating HCCM application."
-${IQE} oc deploy -t templates -s hccm -e ${DEPLOY_ENV} hccm
+${IQE} oc deploy -t templates -s hccm -e dev hccm
```

Comment:
I think this should be left as a variable in case somebody wants to run a different set of configs.

Reply:
This was actually prompted by a change from e2e. You no longer provide the file path. Instead you pass in the environment that you want to pull the configs from. 

From Brandon's email:
```
*NOTE* A key change here is how ocdeployer is run ... now instead of
passing a specific file path, e.g. "-e env/ci.yml", to select the ci
environment you will need to use: "-e ci"
```

If you prefer we can create another environment variable that you can set to `dev`, `ci`, etc. and use that here.

Reply:
You can still pass in a file for backward-compatibility.

But if you want `env/ci.yml` to be merged with `templates/hccm/env/ci.yml` you can just pass `-e ci` and it'll take care of it.

Reply:
Thanks @bsquizz!

Reply:
@blentz when I ran it with just the path to `env/ci.yml` I was getting failures that were resolved when I used `-e ci` instead. I'm definitely open to how you want to handle this!

Reply:
@abaiken - For now, let's go with your change as-is. 

It sounds like this stuff is evolving and we probably don't need to spend the effort on making it perfect (only to have e2e change again underneath us. :wink: ).

Long-term, I foresee converting this script to python at some point just because Bash requires a lot of legwork to do some things that Python handles in the stdlib. (e.g. optparse/argparse for one.)